config/development.json

```diff
@@ -15,6 +15,7 @@
 		"remote-free-extensions": true,
 		"settings": false,
 		"shipping-label-banner": true,
+		"subscriptions": true,
```

Comment:
It will probably still work fine without setting `subscriptions` feature flag in the other two config files. But I still think we should set it there as well, just to `false`, for the `core.json` and `plugin.json`.

Reply:
Good eyes! I added the flag in the commit 2c88a2c